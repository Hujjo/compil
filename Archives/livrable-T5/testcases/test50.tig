    let 
        type int1 = int2
        type int2 = int1
    in
    end
