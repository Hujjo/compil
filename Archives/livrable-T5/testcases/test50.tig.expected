2:21 Circular type definition
